Documentation/configuration.md

```diff
@@ -149,7 +149,110 @@ In `/etc/rkt/auth.d/specific-tectonic.json`:
 }
 ```
 
-The result is that when downloading data from `kubernetes.io` we still
-send `Authorization: Bearer common-token`, but when downloading from
-`coreos.com` - `Authorization: Basic Zm9vOmJhcg==`. And for
-`tectonic.com` - `Authorization: Bearer tectonic-token`.
+The result is that when downloading data from `kubernetes.io` `rkt`
+still sends `Authorization: Bearer common-token`, but when downloading
+from `coreos.com` - `Authorization: Basic Zm9vOmJhcg==` (`foo:bar`
+encoded in base64). And for `tectonic.com` - `Authorization: Bearer
+tectonic-token`.
+
+### rktKind: `dockerAuth`
+
+This kind of configuration is used to set up necessary credentials
+when downloading data from docker indices. The configuration files
+should be placed inside either `auth.d` or `docker.d` subdirectories
```

Comment:
Well, nothing comes to my mind actually. I'll remove the docker.d directory. We can always add it back if more docker configuration kinds will be needed.